### PR TITLE
adding myself

### PR DESCRIPTION
**Pull request by leyang95**

### CODE REVIEW

**Comment by KrashLeviathan** [comment]
You have merge conflicts. It's just because so many people are working on the same page. I'll be putting up a how-to sometime tonight on dealing with merge conflicts, so that should help explain what to do with them.

**Comment by KrashLeviathan** [comment]
https://github.com/KrashLeviathan/club-website/blob/add_tutorials_folder_and_merge_conflicts/resources/tutorials/merge_conflicts.md

File: members/members.json [hunk 1]
```diff
@@ -45,5 +45,9 @@
       "description": "Junior in Computer Engineering"
 
     }
+    "LeYang Loh": {
+      "image": "../_images/members/loh-leyang-md.jpeg",
+      "description": "Junior in Computer Science"
+    },
```
Comment (KrashLeviathan) [inline members/members.json]: The comma here is throwing a wrench in things. Add the comma after the bracket in line 47 instead.

**Comment by KrashLeviathan** [comment]
@leyang95 are you still planning on completing this PR?

**Comment by frostbyte100** [comment]
I'm gonna close this, because I'm assuming he won't or he will tell us eventually.

**Pull request closed without merging**